account/my_auth.py

```diff
@@ -0,0 +1,33 @@
+from .models      import Account
+from my_settings import SECRET
+
+from django.http  import HttpResponse
+from django.http  import JsonResponse
+
+import jwt
+
+def Authorization(func):
+    def wrapper(self, request, *args, **kwargs):
+        token = request.headers.get('Athorization',None)
+
+        if token:
+
+            try:
+                user_id = jwt.decode(token, SECRET['secret']).get('user_id', None)
+                
+                if Account.objects.filter(id = user_id).exists():
+                    user = Account.objects.get(id = user_id)
+                    request.user = user
+
+            except jwt.InvalidTokenError:
+
+                return JsonResponse({'message': 'InvalidToken'}, status = 403)
```

Comment:
CamelCase 보다는 단어사이에 언더스코어로 구분해 주세요
```suggestion
                return JsonResponse({'message': 'INVALID_TOKEN'}, status = 403)
```